src/ITR-examples/ITR_DV.py

```diff
@@ -48,10 +51,9 @@
 # from ITR.configs import LoggingConfig
 
 import osc_ingest_trino as osc
+
 from ITR.data.vault_providers import (
     VaultCompanyDataProvider,
-    VaultProviderProductionBenchmark,
```

Comment:
We should not be able to do ITR calculations without Production and Intensity benchmarks.  Perhaps other linting/comments has turned things off?  Or perhaps I'm such a wizard that I'm doing all the calcs in pure SQL and no longer need those refs in Python.  Nevertheless, they provide functions that should be needed to make the API complete.

Reply:
I took a look at this code.  Here's the cheat:

```
base_production_bm = BaseProviderProductionBenchmark(production_benchmarks=prod_bms)
```

In a future version it should use `VaultProviderProductionBenchmark` but for now we just read in the file.  So let's keep `VaultProviderProductionBenchmark` and let the warning remind us that the code is not yet finished.  Unless warnings screw up pre-commit, in which case leave it commented out.